test/common/alerts_exist.go

```diff
@@ -239,6 +239,157 @@ var expectedRules = []alertsTestRule{
+			"RHMIMiddlewareMonitoringOperatorGrafanaServiceEndpointDown",
+			"RHMIMiddlewareMonitoringOperatorPrometheusOperatedServiceEndpointDown",
+			"RHMIMiddlewareMonitoringOperatorPrometheusServiceEndpointDown",
+			"RHMIMiddlewareMonitoringOperatorRhmiRegistryCsServiceEndpointDown",
+		},
+	},
+	{
+		File: NamespacePrefix + "rhsso-ksm-endpoint-alerts.yaml",
+		Rules: []string{
+			"RHMIRhssoKeycloakServiceEndpointDown",
+			"RHMIRhssoKeycloakDiscoveryServiceEndpointDown",
+		},
+	},
+	{
+		File: NamespacePrefix + "rhsso-operator-ksm-endpoint-alerts.yaml",
+		Rules: []string{
+			"RHMIRhssoKeycloakOperatorRhmiRegistryCsServiceEndpointDown",
+			"RHMIRhssoKeycloakOperatorMetricsServiceEndpointDown",
+		},
+	},
+	{
+		File: NamespacePrefix + "ups-operator-ksm-endpoint-alerts.yaml",
+		Rules: []string{
+			"RHMIUpsOperatorRhmiRegistryCsServiceEndpointDown",
+			"RHMIUpsOperatorUnifiedPushOperatorMetricsServiceEndpointDown",
+		},
+	},
+	{
+		File: NamespacePrefix + "3scale-operator-ksm-endpoint-alerts.yaml",
+		Rules: []string{
+			"RHMIThreeScaleOperatorRhmiRegistryCsServiceEndpointDown",
+			"RHMIThreeScaleOperatorServiceEndpointDown",
+		},
+	},
+	{
+		File: NamespacePrefix + "codeready-workspaces-ksm-endpoint-alerts.yaml",
+		Rules: []string{
+			"RHMICodeReadyCheHostServiceEndpointDown",
+			"RHMICodeReadyDevfileRegistryServiceEndpointDown",
+			"RHMICodeReadyPluginRegistryServiceEndpointDown",
+		},
+	},
+	{
+		File: NamespacePrefix + "user-sso-operator-ksm-endpoint-alerts.yaml",
+		Rules: []string{
+			"RHMIUserRhssoOperatorRhmiRegistryCsMetricsServiceEndpointDown",
+			"RHMIUserRhssoKeycloakOperatorMetricsServiceEndpointDown",
+		},
+	},
+	{
+		File: NamespacePrefix + "apicurito-operator-ksm-endpoint-alerts.yaml",
+		Rules: []string{
+			"RHMIApicuritoOperatorRhmiRegistryCsServiceEndpointDown",
+		},
+	},
+	{
+		File: NamespacePrefix + "solution-explorer-ksm-endpoint-alerts.yaml",
+		Rules: []string{
+			"RHMISolutionExplorerTutorialWebAppServiceEndpointDown",
+		},
+	},
+	{
+		File: NamespacePrefix + "solution-explorer-operator-ksm-endpoint-alerts.yaml",
+		Rules: []string{
+			"RHMISolutionExplorerOperatorRhmiRegistryCsServiceEndpointDown",
+		},
+	},
+	{
+		File: NamespacePrefix + "codeready-workspaces-operator-ksm-endpoint-alerts.yaml",
+		Rules: []string{
+			"RHMICodeReadyOperatorRhmiRegistryCsServiceEndpointDown",
+		},
+	},
+	{
+		File: NamespacePrefix + "fuse-ksm-endpoint-alerts.yaml",
+		Rules: []string{
+			"RHMIFuseOnlineBrokerAmqTcpServiceEndpointDown",
+			"RHMIFuseOnlineSyndesisMetaServiceEndpointDown",
+			"RHMIFuseOnlineSyndesisOauthproxyServiceEndpointDown",
+			"RHMIFuseOnlineSyndesisPrometheusServiceEndpointDown",
+			"RHMIFuseOnlineSyndesisServerServiceEndpointDown",
+			"RHMIFuseOnlineSyndesisUiServiceEndpointDown",
+		},
+	},
+	{
+		File: NamespacePrefix + "ups-ksm-endpoint-alerts.yaml",
+		Rules: []string{
+			"RHMIUpsUnifiedPushServiceEndpointDown",
+			"RHMIUpsUnifiedpushProxyServiceEndpointDown",
```

Comment:
@obrienrobert the naming of these looks a little odd, and I can see why it might from the service names of `ups-unifiedpush` & `ups-unifiedpush-proxy`.
Perhaps `RHMIUPSUnifiedPushServiceEndpointDown` & `RHMIUPSUnifiedPushProxyServiceEndpointDown`

Reply:
@david-martin agreed. I've updated the names of the `UPS` alerts.